UpFlux.Gateway.Server/UpFlux.Gateway.Server/Models/UpdatePackage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace UpFlux.Gateway.Server.Models
+{
+    /// <summary>
+    /// Represents an update package received from the cloud.
+    /// </summary>
+    public class UpdatePackage
+    {
+        /// <summary>
+        /// Gets or sets the unique identifier of the update package.
+        /// </summary>
+        public string PackageId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the version of the update package.
+        /// </summary>
+        public string Version { get; set; }
+
+        /// <summary>
+        /// Gets or sets the filename of the update package.
+        /// </summary>
+        public string FileName { get; set; }
+
+        /// <summary>
```

Comment:
I am not sure where this would be used. Could you please elaborate? Thanks.

Reply:
`File Path`

Reply:
When the update Package comes from the cloud network stream, It will be placed in a temorary storage in the gateway server which i named FilePath.